dungeon.py

```diff
@@ -1,281 +1,94 @@
-#!/usr/bin/env python
-# -*- coding: utf-8 -*-
```

Comment:
???

Reply:
Whoops, forgot about it in the new file. But now that I think about it, is the shebang really necessary in files that are not *main.py*? I mean, it's the only file that should be executed anyway.

Reply:
it doesn't hurt to have it, but the coding should always be specified